redash/settings/__init__.py

```diff
@@ -142,7 +142,7 @@
 # for more information. E.g.:
 CONTENT_SECURITY_POLICY = os.environ.get(
     "REDASH_CONTENT_SECURITY_POLICY",
-    "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-eval'; font-src 'self' data:; img-src 'self' http: https: data: blob:; object-src 'none'; frame-ancestors 'none'; frame-src redash.io;",
+    "default-src 'self'; style-src 'self' 'unsafe-inline'; script-src 'self' 'unsafe-eval' *.segment.com *.segment.io; connect-src 'self' *.segment.com *.segment.io; font-src 'self' data:; img-src 'self' http: https: data: blob:; object-src 'none'; frame-ancestors 'none'; frame-src redash.io;",
```

Comment:
I'm still seeing CSP errors for Hotjar, but I don't actually see where that's coming from in this repo. Should I add `*.hotjar.com` to this header?

Reply:
Yes. It's because the Segment-to-Hotjar integration is client-side.